models/studentDataModel.js

```diff
@@ -16,13 +16,7 @@ var studentSchema = mongoose.Schema({
 	archived: Boolean
 });
 
-var userSchema = mongoose.Schema({
-	username: String,
-	password: String
-});
-
 module.exports.attendace = mongoose.model('Attendace', dataSchema);
 module.exports.grades = mongoose.model('Grades', dataSchema);
 module.exports.data = mongoose.model('Data', dataSchema);
```

Comment:
not really part of this commit, but lets separate the data and student model into two separate files, since they dont rely on each other anymore and dont need to be together